src/components/TopBar/TopBar.vue

```diff
@@ -358,6 +359,7 @@ export default {
 		text-overflow: ellipsis;
 	}
 	.description {
+		font-size: var(--font-size-small, 13px);
```

Comment:
Shouldn't the fallback here be the --default-font-size?

Reply:
You're right. adjusted. Here's the diff from stable29:

![image](https://github.com/user-attachments/assets/49d90c40-acba-4bcb-91a2-288a19e884d5)

Header is thinner now by 7px, but still looks decent (and only desktop with older versions will be affected)